docs/docs/troubleshooting/connection-issues.mdx

```diff
@@ -60,28 +37,18 @@ include:
     shield: settings_reset
 ```
 
-Save the file, commit the changes and push them to GitHub. Download the new firmware zip file build by the latest GitHub Actions job. In it you will find an additional `settings_reset` UF2 image file.
+If you are building for an onboard controller keyboard that doesn't use a shield, you can add a new `shield:` field with its `board:` entry.
```

Comment:
```suggestion
If you are building for an onboard controller keyboard that doesn't use a shield, add the `shield: settings_reset` field to its `board:` entry.
```

Reply:
```suggestion
If you are building for an onboard controller keyboard that doesn't use a shield, add a new `shield:` field to its `board:` entry.
```
I think it's better to keep "a new" rather than "the" to signify the previous non-existence of the field?